GVRf/Sample/gvr-litshader/.project

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>gvrcubemap</name>
```

Comment:
I think you copied the project from gvrcubemap and then modified it. This property needs to be updated otherwise Eclipse will consider this one is the "same" as gvrcubemap.